### PR TITLE
docs(beginner-guide): afterlanding

### PR DESCRIPTION
**Pull request by Awemeter**
Adds on the ground/after landing the part of the beginner's guide

### CODE REVIEW

File: docs/beginner-guide/after-landing-steps.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ![Right Side Overhead Panel positions](../beginner-guide/placeholder.jpg)
+
+=== "Turning into gate"
+
+    * When turning into the gate turn off your nose wheel light and your runway turn off lights
+
+    ![Right Side Overhead Panel](../beginner-guide/placeholder.jpg)
+
+
+    * Continue with the [Powering Down](TBD.MD) steps
```
Comment (Awemeter) [inline docs/beginner-guide/after-landing-steps.md]: Whoever makes the powering down page you will need to change TBD.MD to the file name

Comment (Valastiri) [reply docs/beginner-guide/after-landing-steps.md]: This is actually not needed. The mkdocs has a next page function built in.

Comment (Awemeter) [reply docs/beginner-guide/after-landing-steps.md]: Ok

File: docs/beginner-guide/after-landing-steps.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+### Lower Pedestal
+
+The following actions should be preformed on the lower pedestal
+
+=== "Left Side Lower Pedestal"
+
+    * Set your radio frequency to the one assigned by atc if using atc
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
   * Set your radio frequency to the one assigned by ATC
     * ATC is set in accordance with airport requirements.
```

File: docs/beginner-guide/after-landing-steps.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### Overhead Panel
+
+The following actions should be preformed on the overhead panel and top ecam
+
+=== "Left Side Overhead Panel"
+
+    Set your exterior lights
+
+    !!! info "Exterior Lights"
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]: The info block is probably not needed here. You haven't used it anywhere else and the bullet points should suffice.

File: docs/beginner-guide/after-landing-steps.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    * Start your APU by clicking the APU master button then the APU start button
+
+    * Turn your ati-ice off if not needed
+
+    * Turn the brake fan on if you get a hot brakes ecam warning
+
+    ![Right Side Overhead Panel positions](../beginner-guide/placeholder.jpg)
+
+=== "Turning into gate"
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
=== "Turning Into Gate"
```

File: docs/beginner-guide/after-landing-steps.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+        Your landing lights should look like this:
+
+        ![landing light positions](../beginner-guide/placeholder.jpg)
+
+=== "Right Side Overhead Panel"
+
+    * Start your APU by clicking the APU master button then the APU start button
+
+    * Turn your ati-ice off if not needed
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
    * Turn off your anti-ice if it is not required
```

**Comment by Valastiri** [comment]
Ensure all acronyms are capitalized. i.e. ECAM, ATC, etc.

File: docs/beginner-guide/after-landing-steps.md (hunk 1 above)
Comment (Valastiri) [reply docs/beginner-guide/after-landing-steps.md]: Can you do `[Powering Down](#)` just so there's no strange build errors

File: docs/beginner-guide/after-landing-steps.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
+### Lower Pedestal
+
+The following actions should be preformed on the lower pedestal
+
+=== "Left Side Lower Pedestal"
+
+    * Set your radio frequency to the one assigned by ATC
+        * ATC is set in accordance with airport requirements
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
        * ATC is set in accordance with airport requirements
        
        ![Com panel](../assets/beginner-guide/after-landing-steps/landingLayer-3.png)
```

File: docs/beginner-guide/after-landing-steps.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+    * Disarm the spoilers by pushing down on the lever
+
+    * Set your ENG mode selector to NORM
+
+        ![Com panel](../assets/beginner-guide/after-landing-steps/landingLayer-3.png)
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]: Remove this after moving it.

File: docs/beginner-guide/after-landing-steps.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+        ![Eng knob ](../assets/beginner-guide/after-landing-steps/lgLayer-3.png)
+
+=== "Right Side Lower Pedestal"
+
+    * Retract your flaps to ZERO
+
+    * Set your TCAS to STANDBY
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md:23]:
```suggestion
    * Set your TCAS to STANDBY

        ![Tcas](../assets/beginner-guide/after-landing-steps/landingLayer-4.png)
```

File: docs/beginner-guide/after-landing-steps.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+    * Set your radar OFF
+
+    * Set your predictive windshear system OFF
+
+        ![Tcas](../assets/beginner-guide/after-landing-steps/landingLayer-4.png)
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]: Remove this after moving

File: docs/beginner-guide/after-landing-steps.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+        (If you are crossing runway keep them ON until you vacate the runway)
+
+    * For landing lights, you can choose to retract them or turn them off
+
+    Your lights should look like this
+    ![landing light positions](../assets/beginner-guide/after-landing-steps/After landing-Lights.png)
+
+    or this
+
+    ![landing light positions2](../assets/beginner-guide/after-landing-steps/After Landing-Taxi  Light .png)
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
    Your lights should look like one of the following:

    ^^**Lights Setup 1**^^

    ![landing light positions](../assets/beginner-guide/after-landing-steps/After landing-Lights.png)

    ^^**Lights Setup 2**^^

    ![landing light positions2](../assets/beginner-guide/after-landing-steps/After Landing-Taxi  Light .png)
```

File: docs/beginner-guide/after-landing-steps.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+    * Set your predictive windshear system OFF
+
+        ![Tcas](../assets/beginner-guide/after-landing-steps/landingLayer-4.png)
+        ![Radar](../assets/beginner-guide/after-landing-steps/lgLayer-2.png)
+
+### Overhead Panel
+
+The following actions should be preformed on the overhead panel and top ECAM
+
+=== "Left Side Overhead Panel"
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
=== "Overhead Panel"
```

I feel like this should just be overhead panel and move the following up here: 

* Start your APU by clicking the APU master button then the APU start button

* Turn OFF your anti-ice if it is not required

Could do those two first then exterior lights.

File: docs/beginner-guide/after-landing-steps.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+    * For landing lights, you can choose to retract them or turn them off
+
+    Your lights should look like this
+    ![landing light positions](../assets/beginner-guide/after-landing-steps/After landing-Lights.png)
+
+    or this
+
+    ![landing light positions2](../assets/beginner-guide/after-landing-steps/After Landing-Taxi  Light .png)
+
+=== "Right Side Overhead Panel"
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
=== "Center Instrument Panel"
```

Remove the APU and anti-ice. Keep brake fans here as it's relevant.

File: docs/beginner-guide/after-landing-steps.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+    ![Hot Brakes](../assets/beginner-guide/after-landing-steps/lgLayer-1.png)
+
+=== "Turning Into Gate"
+
+    * When turning into the gate turn off your nose wheel light and your runway turn off lights
+
+    ![Taxi Light](../assets/beginner-guide/after-landing-steps/Beginner-guide-Taxi-Light.png)
+
+    Continue with the [Powering Down](TBD.MD) steps
```
Comment (Valastiri) [inline docs/beginner-guide/after-landing-steps.md]:
```suggestion
    Continue with the [Powering Down](#) steps
```

File: docs/beginner-guide/after-landing-steps.md (hunk 1 above)
Comment (Valastiri) [reply docs/beginner-guide/after-landing-steps.md]: See updated review

**Review by Valastiri — CHANGES_REQUESTED** [review]
Some formatting and moving of images required. 

Also some section header changes for overhead panel as you have ECAM information in "right side overhead" see suggestions.

You also have a merge conflict.

**Review by Valastiri — APPROVED** [review]

**Comment by Valastiri** [comment]
Reviewed by Dan + Imenes. 

Rebased by me. 

Thanks for the work looks good :)

**Pull request merged**